style/wishlist.css

```diff
@@ -1,7 +1,8 @@
 .authentication {
     display: flex;
 }
-.whishlist-container {
+.whishlist-container ,
+.cart-container {
     margin: 5rem 0rem;
     padding: 5rem 0rem;
 }
```

Comment:
Why are you creating separate classes for applying flex, giving margin & padding
Are you not using component library